File: dotnet/socketd/org.noear.socketd/exception/SocketdTimeoutException.cs
namespace org.noear.socketd.exception;


/**
 * 超时异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketdTimeoutException : SocketdException
{
    public SocketdTimeoutException(String message) : base(message)
    {

    }
}

File: java/socketd/src/main/java/org/noear/socketd/exception/SocketDTimeoutException.java
package org.noear.socketd.exception;

/**
 * 超时异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketDTimeoutException extends SocketDException {
    public SocketDTimeoutException(String message){
        super(message);
    }
}
